src/Writer/MySQL.php

```diff
@@ -111,6 +111,11 @@ public function createConnection(array $dbParams): \PDO
             }
         }
 
+        // for mysql8 remove sql_mode "NO_ZERO_DATE"
+        if (version_compare($this->getVersion($pdo), '8.0.0', '>')) {
+            $pdo->query("SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'NO_ZERO_DATE',''));");
```

Comment:
tohle jsem tu přidal protože mysql8 nedokáže uložit prázdný string do `date` - padalo to i v současné verzi co je teď na produkci

Reply:
respektive dokáže, ale je to defaultně zakázaný

Reply:
ok, ... trvalo mi dlhsie kym so to pochopil ... tak ak chapem spravne, tyka sa to verzie 8+ ... a ak je v CSV empty string tak MySQL si to najprv prevedie na `0000...` ... a az v druhom kroku sa to upravuje cez `SET`, takze tam musi byt to vkladanie `0000...` povolene - inak sa to k `SET` ani nedostane.

Reply:
joo přesně tak :)